src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
-body {
+$primary-color: #00acdc;
+$font-color: #060b35;
+$light-grey: #616070;
+$font-family: 'Roboto', sans-serif;
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  border-radius: 5px;
+  width: 200px;
   margin: 0;
+  padding: 32px 16px 16px;
+  font-family: $font-family;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $font-color;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $light-grey;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  align-items: center;
+  justify-content: space-between;
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.card__review_count {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $font-color;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+  padding-left: 1px;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
```

Comment:
Verify that the image path '../images/star.svg' is correct and that the image file exists at this location.